--- Controller/Checkout/Redirect.php
+++ Controller/Checkout/Redirect.php
@@ -7,9 +7,6 @@
 use Magento\Framework\App\Action\Action;
 use Magento\Framework\App\Action\Context;
 use Magento\Framework\App\Config\ScopeConfigInterface;
-use Magento\Framework\App\CsrfAwareActionInterface;
-use Magento\Framework\App\Request\InvalidRequestException;
-use Magento\Framework\App\RequestInterface;
 use Magento\Framework\Exception\AuthenticationException;
 use Magento\Framework\Exception\InputException;
 use Magento\Framework\Exception\LocalizedException;
@@ -27,7 +24,7 @@
 /**
  * Process customers redirect from Gateway and Webhook update requests
  */
-class Redirect extends Action implements CsrfAwareActionInterface
+class Redirect extends Action
 {
     const STATUS_PAID = 'paid';
     const STATUS_CREATED = 'created';
@@ -136,24 +133,6 @@
     }

     /**
-     * This is incoming webhook request so CSRF doesn't exactly apply
-     *
-     * @return bool
-     */
-    public function validateForCsrf(RequestInterface $request): ?bool
-    {
-        return true;
-    }
-
-    /**
-     * @inheritDoc
-     */
-    public function createCsrfValidationException(RequestInterface $request): ?InvalidRequestException
-    {
-        return null;
-    }
-
-    /**
      * Process order success
      *
      * @param $data
